schema/resources/requestCompilation.json

```diff
@@ -8,6 +8,14 @@
     "checksum": {"type": "string"},
     "format": {"type": "string"},
     "force": {"type": "boolean"},
-    "compiler": {"type": "string"}
+    "compiler": {"type": "string"},
+    "stamp_label": {
+      "description": "Label to use in PS/PDF stamp/watermark",
+      "type": "string"
+    },
+    "stamp_link": {
+      "description": "Link to associate with label in PS/PDF stamp/watermark",
+      "type": "string"
```

Comment:
Excellent; thanks for updating this, too

Reply:
I'll document the rest after I investigate whether the other options (format?) are actually doing something. I know I'm always trying to produce PDF with converter so not sure what format does these days. I know 'compiler' is not implemented. I will be revisiting soon.